haptics/README.md

```diff
@@ -9,6 +9,45 @@ npm install @capacitor/haptics
 npx cap sync
 ```
 
+## Permissions
+
+On Android, you must add the `VIBRATE` permission to your app's
+`AndroidManifest.xml` file.
+
+```xml
+<uses-permission android:name="android.permission.VIBRATE" />
+```
+
```

Comment:
```suggestion
```

So, the plugin is already adding the permissions, so users don't have to add it
https://github.com/ionic-team/capacitor-plugins/blob/main/haptics/android/src/main/AndroidManifest.xml#L3

I think we said that plugins should not add permissions, but I think that should only be the case for "hard" permissions that requires the user to prompt and accept them, but for "soft" permissions such as this one, I think it's better to add them in the manifest instead of making users manually add them.

If you don't agree, then the permission should be removed from the `AndroidManifest.xml`

Reply:
I agree, I just forgot we're able to do this because each plugin has its own manifest